package.json

```diff
@@ -2,9 +2,18 @@
   "private": true,
   "version": "1.0.0",
   "description": "A template for kick starting a Cloudflare Workers project",
-  "main": "index.js",
+  "main": "index.ts",
   "license": "MIT",
   "dependencies": {
-    "@cloudflare/kv-asset-handler": "^0.1.3"
+    "@cloudflare/kv-asset-handler": "^0.1.3",
+    "cloudflare": "^2.9.1"
+  },
+  "scripts": {
+    "build": "webpack",
```

Comment:
would be worth adding `husky` too so this script can run on commit... we'll need `dist/index.ts` to be included for this to work